### DISCUSSION TITLE
I want to use the .ckpt files in huggingface, but I face OSerror.

### DISCUSSION THREAD

**Discussion by zengyuzhi** [lean-dojo/ReProver#55] [Q&A] [answered]
I want to run this line : 
python generator/main.py fit --config generator/confs/cli_lean4_random.yaml --model.ret_ckpt_path PATH_TO_RETRIEVER_CHECKPOINT --data.preds_path PATH_TO_PREDICTIONS_PICKLE

I already downloaded the retriever checkpoint from here https://huggingface.co/kaiyuy/leandojo-pl-ckpts/tree/main and set the ckpt path to local and set the google/byt5-small to local path  in both yaml file from generator and retrieval. But I still face the OSerror: could not locate google/byt5-samll in hugging face, something like this. Is there any ways to fix thix? Thanks!

**Comment by yangky11** [comment]
Please describe the exact steps to reproduce the error in details.

**Comment by yangky11** [comment, marked as answer]
I just updated the evaluation script to use Hugging Face checkpoints instead of PyTorch Lightning checkpoints, which hopefully can eliminate this issue. Please follow the instructions in the new README. Feel free to re-open it if you have other questions.

**Reply by zengyuzhi** [reply]
Thank you !!
